rules/te/te-transliteration.js

```diff
@@ -186,37 +186,20 @@
 			[ 'H', 'ః' ],
 			[ 'I', 'ఈ' ],
 			[ 'J', 'ఝ్' ],
-			[ 'K', 'ఖ్' ],
+			[ 'K', 'క్' ],
 			[ 'L', 'ళ్' ],
 			[ 'M', 'ం' ],
 			[ 'N', 'ణ్' ],
 			[ 'O', 'ఓ' ],
-			[ 'Q', 'క్వ్' ],
+			[ 'P', 'ప్' ],
 			[ 'R', 'ఋ' ],
-			[ 'S', 'శ్' ],
-			[ 'T', 'ట్' ],
+			[ 'S', 'ష్' ],
+			[ 'T', 'థ్' ],
 			[ 'U', 'ఊ' ],
-
```

Comment:
@Dhanushcdivakar Thanks for this PR! Is there a reason why you have removed Telugu numbers from the file?